packages/client/src/containers/CollectionsPage/CollectionsPage.Container.js

```diff
@@ -1,12 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ProductList from '../../components/ProductList/ProductList.component';
-import DropDownViewList from '../../components/ProductListDropDown/DropDownViewList/DropDownViewList.component';
+import './CollectionsPage.style.css';
+import DropDownView from '../../components/CategoriesListDropDown/CategoriesListDropDown.component';
+import { apiURL } from '../../apiURL';
 
 export const CollectionsPage = () => {
+  const [products, setProducts] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [onSelectedCategory, setOnSelectedCategory] = useState('');
+  const [onSelectedSortBy, setOnSelectedSortBy] = useState('');
```

Comment:
These are not event handlers, these are just values.
```suggestion
  const [selectedCategory, setSelectedCategory] = useState('');
  const [sortOrder, setSortOrder] = useState('');
```